test/blackbox-tests/test-cases/path-variables/run.t

```diff
@@ -51,6 +53,5 @@ This form does not exist, but displays an hint:
   $ dune build --root jbuild-invalid @test-dep
   Entering directory 'jbuild-invalid'
   File "jbuild", line 5, characters 16-37:
-  Error: ${dep:generated-file} is not supported in jbuild files.
-  Hint: Did you mean ${path:generated-file} instead?
+  Error: Variable ${dep:generated-file} is only available since version 0.0 of the dune language
```

Comment:
This looks odd. Shouldn't this say `since version 1.0` instead?

Reply:
Whoops, I just substituted the wrong version.